spec/models/vm_or_template_spec.rb

```diff
@@ -471,4 +471,14 @@
       expect(VmOrTemplate.batch_operation_supported?(:migrate, [vm1.id, vm2.id])).to eq(true)
     end
   end
+
+  context ".set_tenant_from_group" do
+    it "assigns the tenant from the group" do
+      tenant = FactoryGirl.create(:tenant, :parent => EvmSpecHelper.create_root_tenant)
+      group = FactoryGirl.create(:miq_group, :tenant => tenant)
```

Comment:
Is there a reason you assigned the group to a new tenant instead of the default tenant?  For my rbac tests, I didn't use the default tenant for clarity.  I can't tell if that's the same reason you did it here.